src/lib/krb5/rcache/ser_rc.c

```diff
@@ -64,15 +64,15 @@ krb5_rcache_size(krb5_context kcontext, krb5_pointer arg, size_t *sizep)
     size_t              required;
 
     kret = EINVAL;
-    if ((rcache = (krb5_rcache) arg)) {
+    if ((rcache = (krb5_rcache)arg) && (rcache->ops != NULL)) {
```

Comment:
We should remove the conditional on rcache->ops here rather than moving it.  We do not expect a null ops pointer, the code would already crash if we saw one, and there is no similar conditional in krb5_rcache_externalize().